charts/kube-prometheus-stack/Chart.yaml

```
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 42.1.0
+version: 42.1.1
```

New feature should raise minor version.

```suggestion
version: 42.2.0
```